src/utilities/paramValidation.ts

```diff
@@ -0,0 +1,10 @@
+import { ExtractParamsFromPath, Route, RouteFlat, Routes } from '@/types'
+
+export function routeParamsAreValid(path: string, route: RouteFlat): boolean {
```

Comment:
This makes the ts error go away until we implement this. 
```suggestion
export function routeParamsAreValid(_path: string, _route: RouteFlat): boolean {
```